src/runtime_src/core/edge/drm/zocl/cu_scu.c

```diff
@@ -228,7 +229,6 @@ void xrt_cu_scu_fini(struct xrt_cu *xcu)
 		DRM_WARN("Failed to terminate SCU pid %d.  Performing SIGKILL.\n",core->sc_pid);
 		kill_pid(p, SIGKILL, 1);
 	}
-	udelay(1);
```

Comment:
Glad to see code being removed.  
Just wondering why was this code here.  Was it to force a context switch or something else?

Reply:
Initially I wasn't sure if kill_pid will only return after process is killed, so added a delay here.
Then realized later this wasn't necessary.